projects/plugins/boost/changelog/update-boost-minify-modules-fixes

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+Boost: update minification modules so they are consistent with the rest of the UI
```

Comment:
It shouldn't say Boost, it should be the aspect of Boost that's being updated. I think it can also mention the features' UI it updated like "Update Page Cache, Concatenate JS/CSS and Image CDN - Image Quality settings to have a more unified look" or something 😅 

I think we talked about this but you might have forgotten to update it.

Reply:
I think that was a different PR, but thanks for repeating it. Changelog updated. I forgot to update it after the scope of this PR widened.